quinn-proto/src/crypto/rustls.rs

```diff
@@ -5,34 +5,38 @@ use std::{
     sync::Arc,
 };
 
-use ring::{aead, hmac};
+use ring::{aead, hkdf, hmac};
 pub use rustls::TLSError;
 use rustls::{
     self,
-    quic::{ClientQuicExt, ServerQuicExt},
+    quic::{ClientQuicExt, Secrets, ServerQuicExt},
     Session,
 };
 use webpki::DNSNameRef;
 
-use super::ring::Crypto;
+use super::ring::{header_key_from_secret, initial_keys, PacketKey};
```

Comment:
I think I would prefer not to important these by name here and just reference them through `ring` to make their provenance more obvious.

Reply:
That runs into awkward collisions with the crate name. We could work around those, but I think it's confusing to read as well.

Reply:
Okay, let's leave it.